src/server.lisp

```diff
@@ -61,8 +61,8 @@
   "Add a function F to DISPATCH-TABLE, for use by an RPCQ server.  The function F is expected to take only keyword arguments.
 
 By default, a symbol passed in for F will be automatically converted into the name of F used in the dispatch table.  To manually specify a name (or to provide a name for a non-symbol value of F), use the keyword argument :NAME."
-  (check-type f function)
   (check-type name string)
+  (alexandria:ensure-function f)
```

Comment:
```suggestion
  (setf f (alexandria:ensure-function f))
```

Reply:
@ecp-rigetti Ah, I misunderstood the meaning here. Ignore this change, but we should check that it names a function appropriately.

```
(defun global-function-p (symbol)
  "Return true if SYMBOL names a global function. Return false if it doesn't."
  (check-type symbol symbol)
  (and (fboundp symbol)
       (not (macro-function symbol))
       (not (special-operator-p symbol))))
```

then do something like
```
(unless (global-function-p f)
  (warn "The symbol ~S doesn't name a global function." f))
```

Ensure a symbol is provided by doing a
```
(check-type f symbol)
```
But you'll also want to make sure it has a definition, or at least warn if it doesn't. Why not do
```
(when (